Client/Frontend/Settings/AppSettingsOptions.swift

```diff
@@ -572,6 +572,7 @@ class SendAnonymousUsageDataSetting: BoolSetting {
             settingDidChange: {
                 AdjustIntegration.setEnabled($0)
                 LeanplumIntegration.sharedInstance.setEnabled($0)
+                LeanplumIntegration.sharedInstance.setUserAttributes(attributes: ["Telemetry Opt Out" : $0])
```

Comment:
Do this before you enable or disable it, as if you disable it, it may not get to Leanplum. Please test this too.